include/phasar/PhasarLLVM/DataFlowSolver/Mono/CallString.h

```diff
@@ -67,7 +68,8 @@ template <typename T, unsigned K> class CallString {
   friend bool operator<(const CallString &Lhs, const CallString &Rhs) {
     return Lhs.cs < Rhs.cs;
   }
-  friend std::ostream &operator<<(std::ostream &OS, const CallString &C) {
+  friend llvm::raw_ostream &operator<<(llvm::raw_ostream &OS,
+                                       const CallString &C) {
     std::copy(C.CS.begin(), --C.CS.end(), std::ostream_iterator<T>(OS, " * "));
```

Comment:
Does std::ostream_iterator really work with llvm::raw_ostream? Or does this only compile, because it is never instantiated?

Reply:
don't know. @pdschubert ?

Reply:
Try it out?
I would think, it's just not instantiated and would not compile otherwise.

Reply:
I have tried it out: ostream_iterator does not work with raw_ostream. 

However, llvm::interleave works very well:
`llvm::interleave(C.CS, OS, " * ");`

Reply:
This solution looks much more readable :+1:

Reply:
Yep, looks good ;-)